### ISSUE TITLE
UI Tweaks

### ISSUE THREAD

**Issue by test2a** [elvistony/jio-pyserver#10]
i can see the code for EPG but i can't seem to go past animal planet. is there a way we can have the EPG or the channel list at least?

**Comment by mayankwadhwani** [comment]
channel list is present at path /channelList Ref : https://github.com/elvistony/jio-pyserver/blob/d0efcd2964b5c13d00b7e229461380c6fb330c53/app.py#L38

**Comment by test2a** [comment]
@mayankwadhwani i saw that. 

127.0.0.1:1024/channelList

this worked, i was trying "127.0.0.1:1024/ChannelList.html"

maybe put this in the readme because its not exactly easy to find?

**Comment by elvistony** [comment]
Thanks @mayankwadhwani for filling in, Yeah that's my bad.  Feel free to raise Pull requests with fixes that ya'll can conjure.

**Comment by test2a** [comment]
Why is the stream choppy?  I am on a 100mbps plan so I should not have any buffering issues. Has anyone else noticed that? Is it downloading each chunk one by one, displaying and waiting for the next ?

**Comment by elvistony** [comment]
Feel free to raise PRs with fixes if you could figure out the problem.

**Comment by test2a** [comment]
@elvistony i have an idea. can we have the channel list as the default instead of a single channel?  and improve the channel page so that is better on the phone for example and has search/filters. 

currently the default channel is not easy to change and typing the /channelList on the phone is not easy.

also, would it be possible to show the channellist button on the video overlay?

**Comment by elvistony** [comment]
Yeah some of these I've already changed in my local repo, I'll make these changes and push them soon. Kinda busy these days, might take a while :( .

**Comment by test2a** [comment]
dont worry. this isnt mission critical stuff. i was looking at the resource usage and it is around 50+30mb for python3 on my laptop. now i am thinking of using the pi zero that i have for pihole to serve tv. then use duckdns to serve that over a subdomain. or, i could just use any of my vps servers to do that. this is really good

**Comment by elvistony** [comment]
Woah, what are the odds... I just launched my first pihole few days ago. But due to the CPU usage, the script never served the video from that pi.
Kinda curious, what's use of duckdns?

**Comment by test2a** [comment]
@elvistony i've  been running pihole on my pi zero since jio first launched in my area, around 1.5~2 years ago. i occasionally ssh into it to update it but otherwise, i have plugged it in the usb port of the jio router.

are you using pi zero or some other pi to serve videos? i do suppose the zero has very less data and IO bandwidth for video, thats understandable.


duckdns is to access your network from anywhere. last year i was having trouble with a software and i did 

Rpi>duckdns>jio router port forward and disable firewall > SSH from sweden or france i'm not sure.


then one time i hosted a small project on the local pi 3b+, used duckdns to give it internet access and created a subdomain to access that project. 

https://github.com/test2a/JioFiberwebhosting

havent checked this since the last commit but i have had a lot of fun with duckdns

**Comment by elvistony** [comment]
That sounds awesome, I tried port forwarding on my ISP (local) they have their subscribers behind a NAT, so can't route anything outside without a vpn.

Will definitely checkout your repo tho, looks interesting!

**Comment by test2a** [comment]
@elvistony

https://user-images.githubusercontent.com/39449563/160580853-fda4ca64-4421-4823-bfe8-6d545992c550.mp4


https://user-images.githubusercontent.com/39449563/160580884-ac0b4ec8-bdae-483d-b893-dc261f931c38.mp4



a couple of inputs. 

1. the default channel list is much more accessible. thank you for that.
2.  "some" channels do not seem to work. i first thought it was something to do with HD channels but then found colors HD works just great so some channels dont work and i dont know why. 
3.  can we make the mobile table responsive? 
4. would it be possible to add channel filter? https://www.w3schools.com/howto/howto_js_filter_table.asp i tried myself but i could not.
5. on the channel page, we have a "play stream" button, can we do a "go back to channel" button?

**Comment by elvistony** [comment]
Thanks for your feedback!

- a2) Yeah the API endpoint that was used have some dead channels, would need some support to consolidate a list to be removed.
- a3) Yeah probably gonna move to bootstrap. 
- a4) I'm working on adding the filters to the channel list, thinking of moving to bootstrap (maybe).
- a5) If you mean to go back to channel list? yeah sure, I'll check that too!

**Comment by test2a** [comment]
@elvistony i forgot to mention it earlier, can you matrix bridge the telegram group?
